lib/winrm/winrm_service.rb

```diff
@@ -254,7 +254,12 @@ def get_command_output(shell_id, command_id, &block)
           next if n.text.nil? || n.text.empty?
 
           # decode and strip off BOM which win 2008R2 applies
-          stream = { n.attributes['Name'].to_sym => Base64.decode64(n.text).force_encoding('utf-8').sub("\xEF\xBB\xBF", "") }
+          decoded_text = Base64.decode64(n.text).force_encoding('utf-8')
```

Comment:
It seems that instead of doing a validity check, we could just change the `force_encoding` to `encode` and secify `:invalid` and `:udef` options:
```
stream = { n.attributes['Name'].to_sym => Base64.decode64(n.text).encode('utf-8', invalid: :replace, undef: :replace).sub("\xEF\xBB\xBF", "") }
```